frontend/cypress/e2e/agreementList.cy.js

```diff
@@ -16,7 +16,9 @@ it("loads", () => {
 });
 
 it("navigates to the approveAgreements page when the approve button is clicked", () => {
+    cy.wait(1000);
```

Comment:
Just curious, can we reduce this any? Having it at all is annoying, but would like to see if we can find a minimum amount required (500ms?). No biggie if not.

Reply:
removed it and it still works